src/SIL.XForge.Scripture/ClientApp/src/app/translate/draft-generation/draft-generation-steps/draft-generation-steps.component.html

```diff
@@ -179,14 +179,77 @@ <h1 class="mat-headline-4">{{ t("configure_advanced_settings") }}</h1>
               class="advance-button"
               [disabled]="isStepsCompleted"
             >
-              {{ t("generate_draft") }}
+              {{ t("next") }}
             </button>
             @if (isStepsCompleted) {
               <mat-spinner diameter="24"></mat-spinner>
             }
           </div>
         </mat-step>
       }
+      <mat-step [completed]="true">
+        <ng-template matStepLabel>
+          {{ t("confirm_label") }}
+        </ng-template>
+        <h1 class="mat-headline-4">{{ t("confirm") }}</h1>
+
+        <h2>{{ t("confirm_training_title") }}</h2>
+        <mat-card class="confirm-section mat-elevation-z2">
+          <span class="explanation">{{ t("confirm_training") }}</span>
+          <table mat-table class="confirm-training" [dataSource]="selectedTrainingBooksCollapsed()">
+            <ng-container matColumnDef="book">
+              <th mat-header-cell *matHeaderCellDef></th>
+              <td mat-cell *matCellDef="let element" class="bookName">
+                <div class="confirm-books cell-padding-block">
+                  @for (range of element.ranges; track range) {
+                    @if (range !== element.ranges[element.ranges.length - 1]) {
+                      <span class="confirm-book">{{ range }}, </span>
```

Comment:
This should be updated to use `i18n.enumerateList`

Reply:
Done. We do lose out on the word-wrapping control we had previously, however